include/GafferSceneUI/SceneGadget.h

```diff
@@ -150,7 +150,9 @@ class GAFFERSCENEUI_API SceneGadget : public GafferUI::Gadget
 		/// Finds the path of the frontmost object intersecting the specified line
 		/// through gadget space. Returns true on success and false if there is no
 		/// such object.
+		/// hitPoint is the approximate intersection point in gadget space
```

Comment:
Move comment to function it applies to. Suggest :

As above. Additionally `hitPoint` is filled with the approximate intersection point in gadget space.

Reply:
Reworded in https://github.com/GafferHQ/gaffer/pull/3403/commits/6ad9d5158f888868df09d5b1451e4e241f4a9654